src/components/auth/Register.js

```diff
@@ -3,36 +3,35 @@ import styled from "styled-components";
 import { Container, Form, Button, Col } from "react-bootstrap";
 import MainLogo from "../../assets/raise_logo_background.png";
 import Pinwheel from "../../assets/pinwheel.png";
-import Stepper from "../standard/Stepper"
+import Stepper from "../standard/Stepper";
+import userPool from "./poolData";
+import { CognitoUser, CognitoUserAttribute } from "amazon-cognito-identity-js";
```

Comment:
Nit: Let's put this at the top above react-bootstrap, so our module imports are all together